browser/src/control/Control.Notebookbar.js

```diff
@@ -555,22 +555,31 @@ L.Control.Notebookbar = L.Control.extend({
 	},
 
 	getOptionsSectionData: function() {
-		return this.buildOptionsSectionData([
+		return this.buildOptionsSectionData(this.getDefaultToolItems());
+	},
+
+	getDefaultToolItems: function() {
+		return [
 			{
 				'type': 'toolitem',
 				'text': _UNO('.uno:Sidebar', '', true),
-				'command': '.uno:SidebarDeck.PropertyDeck'
+				'command': '.uno:SidebarDeck.PropertyDeck',
+				'accessibility': { focusBack: false,	combination: 'SB', de: null }
 			},
 			{
 				'type': 'toolitem',
 				'text': _UNO('.uno:Navigator'),
-				'command': '.uno:Navigator'
+				'command': '.uno:Navigator',
+				'accessibility': { focusBack: false, combination: 'K', de: 'V' }
 			},
 			{
-				'type': 'toolitem',
-				// dummy node to avoid creating labels
+				'type': 'customtoolitem',
+				'text': _('Share'),
+				'command': 'shareas',
+				'accessibility': { focusBack: false, combination: 'SH', de: null },
```

Comment:
I would prefer to group all sidebar buttons in terms of access-key by assign the same first letter and the second letter should be based on the label and not the uno command. So, in this case would be : `ZS`